bodhi/tests/server/test_schemas.py

```diff
@@ -21,7 +21,7 @@
 from bodhi.server import schemas
 
 
-class TestSchemas(unittest.TestCase):
+class TestSchemas():
```

Comment:
In the interests of consistency (and because you have sign-off on your commit anyway), would you remove the parens?, I.e.:

````
class TestSchemas:
````